app/cdap/components/RecipeManagement/RecipeDetails/DetailContainer/index.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Typography } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import T from 'i18n-react';
+import { IRecipe } from 'components/RecipeList/types';
+import { format, TYPES } from 'services/DataFormatter';
+import DirectiveTable from 'components/RecipeManagement/RecipeDetails/DirectivesTable';
+
+interface IRecipeDetailsProps {
+  selectedRecipe: IRecipe;
+}
+
+const PREFIX = 'features.WranglerNewUI.RecipeDetails';
+
+const DetailsWrapper = styled(Box)`
+  margin-top: 19px;
+`;
+
+const StepsGridHead = styled(Typography)`
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 150%;
+  color: ${grey[700]};
+`;
+
+const StyledRecipeName = styled(StepsGridHead)`
+  text-transform: capitalize;
```

Comment:
can we rename this to RecipeName
we don't need to use the Styled keyword unless the output component name is the same as the input component while styling
support const Button = styled(Button)``, then we can use the relative name to Button output i.e. PrimaryButton, SecondaryButton, TertiaryButton, or some custom name relative to the expected behavior of the Button, otherwise as a final resort we can use StyledButton
please check this across the PR changes